docs/ManualDeploymentGuideScalarDLOnAzure-1.md

```diff
@@ -0,0 +1,217 @@
+
+![image](images/azure-diagram.png)
+
+In this guide, we will create the following components.
+
+* An Azure Virtual Network associated with a Resource Group.
+* An AKS cluster with 2 node pools.
+* A managed database service.
+    * A Cosmos DB Account.
+* A Bastion instance with a public IP.
+* Azure container insights.
+
+## Prerequisites
+
+You must have required permissions as specified in [Cloud permissions document](./CloudPermissionsForScalarDLOnAKS.md) to create cloud resources to deploy Scalar DL on Azure.
+
+## Step 1. Configure your network
+
+Configure a secure network with your organizational or application standards.
+Scalar DL handles highly sensitive data of your application, so you should create a highly secure network for production.
+This section shows how to configure a secure network for Scalar DL deployments.
+
+### Requirements
+
+* You must create a virtual network with a subnet for bastion.
+* You must create 2 subnets with the prefix of at least `/22` for AKS, one subnet must be created with the name `k8s_ingress` to create an envoy load balancer.
+
+### Recommendations
+
+* You should create a bastion server to manage the Kubernetes cluster for production.
+
+### Steps
+
+* Create an Azure virtual network on the basis of [Azure official guide](https://docs.microsoft.com/en-us/azure/virtual-network/quick-create-portal) with the above requirements and recommendations.
+* Create a bastion server on the basis of [Azure official guide](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/quick-create-portal).
+
+## Step 2. Set up a database
+
+In this section, you will set up a database for Scalar DL.
+
+### Requirements
+
+* You must have a database that Scalar DL supports.
+
+### Steps
+
+* Follow [Set up a database guide](./SetupDatabase.md) to set up a database for Scalar DL.
+
+## Step 3. Configure AKS
+
+This section shows how to configure a Kubernetes service for Scalar DL deployment.
+
+### Prerequisites
+
+Install the following tools on the bastion for controlling the AKS cluster:
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli): In this guide, Azure CLI is used to create a kubeconfig file to access the AKS cluster.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/): Kubernetes command-line tool to manage AKS cluster. Kubectl 1.19 or higher is required.
+
+### Requirements
+
+* You must have an AKS cluster with Kubernetes version **1.19** or higher for Scalar DL deployment.
+* You must create a new `user node pool` with the name `scalardlpool` for Scalar DL deployment.
+* You must select the subnet other than `k8s_ingress` subnet for creating the AKS cluster.
+* You must create a Kubernetes cluster with `service principal` as the Authentication method.
+* You must create a Kubernetes cluster with `Azure CNI`.
+* You must add a role assignment to the `k8s_ingress` subnet to access the newly created AKS cluster service principal using the role of `Network Contributor`.
```

Comment:
Do we need to configure the service endpoint?
https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-configure-vnet-service-endpoint

Reply:
Yes, I have added a recommendation in database guide to configure service endpoint in 805dc17